darwin/utils/utils.py

```diff
@@ -764,19 +764,16 @@ def _parse_darwin_raster_annotation(annotation: dict) -> Optional[dt.Annotation]
 
 
 def _parse_darwin_mask_annotation(annotation: dict) -> Optional[dt.Annotation]:
-    if not annotation.get("mask"):
-        raise ValueError("Mask annotation must have a 'mask' field")
-
     id: Optional[str] = annotation.get("id")
     name: Optional[str] = annotation.get("name")
     mask: Optional[dt.JSONFreeForm] = annotation.get("mask")
     slot_names: Optional[List[str]] = parse_slot_names(annotation)
 
-    if not id or not name or not mask or not slot_names:
+    if not id or not name or mask == None or not slot_names:
```

Comment:
mask field could contain empty object which is valid, but is falsy value, therefore we should explicitly check None value.